pkg/check/binlog.go

```diff
@@ -0,0 +1,161 @@
+package check
+
+import (
+	"database/sql"
+	"fmt"
+	"strings"
+
+	"github.com/juju/errors"
+	"github.com/ngaut/log"
+	"github.com/pingcap/tidb-tools/pkg/dbutil"
+)
+
+// MySQLBinlogEnablePreChecker checks whether `log_bin` variable is enabled in MySQL.
+type MySQLBinlogEnablePreChecker struct {
```

Comment:
rename `MySQLBinlogEnablePreChecker` to `MySQLBinlogEnableChecker`, and the same to `MySQLBinlogFormatPreChecker`, `MySQLBinlogRowImagePreChecker`, etc.